sdks/python/apache_beam/io/textio_test.py

```diff
@@ -641,6 +654,82 @@ def test_read_bzip2_concat(self):
       assert_that(lines, equal_to(expected))
       pipeline.run()
 
+  def test_read_deflate(self):
+    _, lines = write_data(15)
+    with TempDir() as tempdir:
+      file_name = tempdir.create_temp_file()
+      with open(file_name, 'wb') as f:
+        f.write(zlib.compress('\n'.join(lines)))
+
+      pipeline = TestPipeline()
+      pcoll = pipeline | 'Read' >> ReadFromText(
+          file_name,
+          0, CompressionTypes.DEFLATE,
+          True, coders.StrUtf8Coder())
+      assert_that(pcoll, equal_to(lines))
+      pipeline.run()
+
```

Comment:
Please add tests for text sink (textio.WriteToText) as well.

Reply:
+1 Thanks for catching that. Added.